sympy/sets/tests/test_sets.py

```diff
@@ -1046,6 +1046,16 @@ def test_issue_10931():
     assert S.Integers - S.Reals == EmptySet()
 
 
+def test_issue_11442():
+    a = Symbol('a', real=True)
+    b = Symbol('b', real=True)
+    i1 = Interval(2, 7)
+    i2 = Interval(a, b)
+    assert not i1.is_EmptySet
```

Comment:
I know it, but it failed. i1.is_EmptySet gives us False, but that False doesn't equal False... It was strange for me and I changed test for not